docs/sites/site_navigation.md

```diff
@@ -36,26 +36,21 @@ Or you can add one of your [pages](../sites/site_pages), which should be listed
 
 ![URL block with the label “RRCHNM” and url chnm.gmu.edu](../sites/sitesfiles/sitenav_url.png) URL Block
 
-**Browse** blocks create a browse page for [resources](../sites/site_resources.md) already associated with your site. There are blocks to create a browse items page and a block to browse item sets.  
+**Browse** blocks create a browse page for [resources](../sites/site_resources.md) already associated with your site. There are blocks to create a browse items page and a block to browse item sets page.  
 
 You can change the *label* of the block to reflect what, specifically, site visitors will browse. 
 
-If you want to refine the Browse block, you can use the Query to create a search query that returns the desired results. 
+![Browse block with the query element](../sites/sitesfiles/sitenav_query.png)
 
-![Browse block with the label ‘Browse’ and no query](../sites/sitesfiles/sitenav_browse.png)
+To refine the Browse block, you can use the Query to create a search query that returns the desired results. Note that you **must** enter a query for the block to function.
 
-Note that you **must** enter a query for the block to function.
+Click the Edit button to reveal the advanced search interface in the drawer. Once you are satisfied with the universe of items in your search, clicking the Apply button will set the universe of items for the Browse block.
```

Comment:
I'm not sure about the phrase "universe of items" here and other places below. It's accurate, but I'd reserve "universe" for the entire pool of items available to all sites. It may be clearer to say "delineation of items" or "pool of items."